dali/python/nvidia/dali/pipeline.py

```diff
@@ -1528,23 +1603,27 @@ def my_pipe():
 
         pipe = my_pipe(batch_size=42, num_threads=3)
         ...
+
+    Keyword args
+    ------------
+
+    enable_conditionals : bool, optional
```

Comment:
How about `enable_cond`? I know it's not that well descriptive than `enable_conditionals`, but my thinking here is that I never know, whether I should use singular or plural... `enable_conditional` or `enable_conditionals`?

Reply:
Wouldn't this issue be better solved by IntelliSense/syntax support in Python instead? When possbile, I am up for making things as descriptive as possible. But this is just a thought, I am fine either way